test/controllers/sessions_controller_test.rb

```diff
@@ -38,7 +38,6 @@ class SessionsControllerTest < ActionController::TestCase
     test "#new should render a full-page if the shop param doesn't exist" do
       get :new
       assert_response :ok
-      assert_template :new
```

Comment:
fine with me. Perhaps we can add an assertion on the contents of response.body to replace this